PolyEngine/Engine/Src/DeferredTaskSystem.cpp

```diff
@@ -20,13 +20,6 @@ void DeferredTaskSystem::DeferredTaskPhase(World* w)
 	}
 }
 
-UniqueID DeferredTaskSystem::SpawnEntityImmediate(World* w)
```

Comment:
I didn't ask you to remove `SpawnEntityImmediate` Why is it removed? I asked only to remove `SpawnEntityDeferredTask` class.